### PR TITLE
context and redux

### PR DESCRIPTION
**Pull request by niharikadayama**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,17 +13,17 @@
...
-    "native-base": "^3.2.1",
     "react": "17.0.2",
     "react-native": "0.65.1",
     "react-native-gesture-handler": "^1.10.3",
     "react-native-reanimated": "^2.2.3",
     "react-native-safe-area-context": "^3.3.2",
     "react-native-screens": "^3.8.0",
     "react-native-vector-icons": "^8.1.0",
-    "react-navigation": "^4.4.4"
+    "react-navigation": "^4.4.4",
```
Comment (Parth-coditas) [inline package.json:24]: There are multiple versions of **react-navigation** being used.

```
"@react-navigation/native": "^6.0.6",
"react-navigation": "^4.4.4",
```

File: src/Screens/wishlist/index.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+const Wishlist = ({navigation,route}) =>{
+    const item = route.params
+    
+    return (
+    <View style={styles.container}>
+        
+           <Header leftIcon={"arrow-left"} rightIcon={"shopping-cart"} size={28}
+                onLeftIconPress ={()=>{ navigation.navigate('details') }}
+                onRightIconPress ={()=>{ navigation.navigate('') }}
+                image= {false} />
```
Comment (Parth-coditas) [inline src/Screens/wishlist/index.tsx:16]: you can use defaultProps to pass a default value to any prop.

File: src/components/wishlistCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import {View,Text,Image} from 'react-native'
+import Icon from 'react-native-vector-icons/FontAwesome'
+import styles from "./styles";
+import {Button} from 'components'
+import { NavigationContainer } from "@react-navigation/native";
+
+const WishlistCard = ({item}) =>{
```
Comment (Parth-coditas) [inline src/components/wishlistCard/index.tsx:8]: You haven't defined what will be the type of the item Prop.

**Review by Parth-coditas — CHANGES_REQUESTED** [review]

File: src/Screens/wishlist/index.tsx (hunk 2 above)
Comment (niharikadayama) [reply src/Screens/wishlist/index.tsx:16]: yes but all the icons have specific role therefore I have not used that

File: src/components/wishlistCard/index.tsx (hunk 3 above)
Comment (niharikadayama) [reply src/components/wishlistCard/index.tsx:8]: I will update after removing all the extra provers and other declarations.

**Pull request merged**